docs/pguidance/agent-install-packages.md

```diff
@@ -0,0 +1,142 @@
+# Install using *nix packages
+
+Install the SignalFx Smart Agent using either a Debian or RPM package,
+and then verify that the SignalFx Smart Agent is working.
+
+## Install the SignalFx Smart Agent using a Debian package
+
+This section describes how to install the Smart Agent on hosts that
+run Linux distributions based on Debian.
+
+### Prerequisites for Debian
+
+* *nix distribution that's based on Debian or supports Debian packages
+* Kernel version 2.6 or higher
+* CAP_DAC_READ_SEARCH and CAP_SYS_PTRACE capabilities
```

Comment:
capital here, lowercase above -- also use backticks

Reply:
Also per https://signalfuse.atlassian.net/browse/DOCS-1560 I think we need to up that kernel version everywhere to 2.6.32

Reply:
I'll fix this.